### PR TITLE
Improve certificate generation makefile

### PR DESCRIPTION
**Pull request by elevran**
- add a root-ca config
- leave intermediate files to save work when creating multiple intermediate certificates

### CODE REVIEW

File: samples/certs/Makefile [hunk 1]
```diff
@@ -56,17 +82,15 @@ root-key.pem:
...
 		-extensions req_ext -extfile $(dir $<)/intermediate.conf \
 		-in $< -out $@
 
 %/cluster-ca.csr: L=$(dir $@)
 %/cluster-ca.csr: %/ca-key.pem %/intermediate.conf
 	@echo "generating $@"
-	@openssl req -new -batch -sha256 $(UTF8) \
-		-config $(L)/intermediate.conf \
-		-key $< -out $@ 
+	@openssl req -new -config $(L)/intermediate.conf -key $< -out $@
```
Comment (vadimeisenbergibm) [inline samples/certs/Makefile]: so why `-batch` is removed?

Comment (vadimeisenbergibm) [reply samples/certs/Makefile]: ^ is it handled by `prompt = no` ?

Comment (elevran) [reply samples/certs/Makefile]: Correct

**Pull request merged**